### DISCUSSION TITLE
The following arguments are required: path_to_hdf5, path_to_csv

### DISCUSSION THREAD

**Discussion by embeddedss** [antonior92/automatic-ecg-diagnosis#28] [Q&A]
Hi  @antonior92,

I'm running into train.py and it is showing this problem.

![屏幕截图(133)](https://user-images.githubusercontent.com/122141002/221558297-cac2811a-8b6e-4f11-80c6-0aa325d58fc8.png)

Can you help me solve it if it is convenient? thank you.

Thanks!

**Comment by antonior92** [comment]
You need to specify the path dataset you are using to train the model as command line argument. You got this error because you did not passed these arguments
